scripting/javascript.rst

```diff
@@ -366,7 +366,7 @@ debug stream may provide some helpful clues.
             0.000000/1685018723.551120 [plugin Zeek::JavaScript] Registered zeek_init
    1685018723.601898/1685018723.621106 [plugin Zeek::JavaScript] ZeekInvoke: invoke for zeek_version
    1685018723.601898/1685018723.621177 [plugin Zeek::JavaScript] Invoke zeek_version with 0 args
-   1685018723.601898/1685018723.621212 [plugin Zeek::JavaScript] ZeekInvoke: invoke for zeek_version returned: 
+   1685018723.601898/1685018723.621212 [plugin Zeek::JavaScript] ZeekInvoke: invoke for zeek_version returned:
```

Comment:
Alternatively move the line below upwards, doesn't seem the line break was wanted.